tests/integration/targets/kubevirt_vm/playbook.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Create VM
+  connection: local
+  gather_facts: false
+  hosts: localhost
+  tasks:
+    - name: Create a VirtualMachine
+      kubernetes.kubevirt.kubevirt_vm:
+        state: present
+        name: testvm3
+        namespace: default
+        labels:
+          app: test
+        wait: true
+        wait_timeout: 600
+        spec:
+          domain:
+            cpu:
+              sockets: 1
+            memory:
+              guest: 1280Mi
```

Comment:
Weird amount of memory.

Reply:
hey that's 1024+256 don't make me feel old!